tests/generator/test_make_fp.py

```
@@ -548,6 +589,21 @@ def _check_pwscf_input_head(testCase, idx):
         testCase.assertEqual(("\n".join(lines)).strip(), pwscf_input_ref.strip())
 
 
+def _check_cpx_input_head(testCase, idx):
+    fp_path = os.path.join("iter.%06d" % idx, "02.fp")  # noqa: UP031
```

Ruff (0.13.3)

Unused `noqa` directive (non-enabled: `UP031`)

Remove unused `noqa` directive

(RUF100)